From babb185b61a4094625ef716519764003a50d4af7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 12 May 2025 10:36:25 +0800
Subject: [PATCH 2/4] DEBIAN: configure.ac: allow building with system flags

---
 configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index f4f6156..705b5db 100644
--- a/configure.ac
+++ b/configure.ac
@@ -127,7 +127,8 @@ else
   ALIGN_FLAGS="-falign-jumps=1 -falign-loops=1 -falign-functions=1"
 fi
 
-CFLAGS=""
+# This disables possibility to specify any additional CFLAGS in configure line
+#CFLAGS=""
 MINIMAL_CFLAGS=""
 GENERIC_CFLAGS=""
 
-- 
2.49.0

