From 776423d348c6531455b98eadf4049bcfc4fea36b Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 19 Jan 2026 09:20:48 +0800
Subject: [PATCH 1/2] AOSCOS: fix(configure.ac): enable detection for Python
 3.14

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6fd5b4bd..b14c7c1f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -614,7 +614,7 @@ if test "$found_py" = "1"; then
 	if test "$found_py" = "1" -o "$found_py3" = "1"; then
 	  # Reset everything, so that we can look for another Python.
           m4_foreach([mym4pythonver],
-                     [[2.7],[3.6],[3.9],[3.10],[3.11],[3.12],[3.13],[all]],
+                     [[2.7],[3.6],[3.9],[3.10],[3.11],[3.12],[3.13],[3.14],[all]],
            [unset PYTHON
 	    unset PYTHON_VERSION
 	    unset PYTHON_CPPFLAGS
-- 
2.52.0

